### PR TITLE
Add instructions output to Slurm controller for SSH command

### PR DESCRIPTION
**Pull request by nick-stroud**
Adding `outputs: [instructions]` to controller will print command to SSH.

### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

**Review by harshthakkar01 — APPROVED** [review]

**Comment by nick-stroud** [comment]
running 1 Slurm integration test

File: community/modules/scheduler/schedmd-slurm-gcp-v6-controller/outputs.tf [hunk 1]
```diff
@@ -21,3 +21,11 @@ output "slurm_bucket_path" {
   description = "Bucket path used by cluster."
   value       = module.slurm_files.slurm_bucket_path
 }
+
+output "instructions" {
+  description = "Post deployment instructions."
+  value       = <<-EOT
+    To SSH to the controller (may need to add '--tunnel-through-iap'):
+      gcloud compute ssh --zone ${var.zone} ${local.controller_hostname} --project ${var.project_id}
```
Comment (mr0re1) [inline community/modules/scheduler/schedmd-slurm-gcp-v6-controller/outputs.tf:29]: nit. you can make it shorter by outputting

```sh
gcloud compute ssh  ${module. slurm_controller_instance .instances_self_links[0]}
```

Comment (nick-stroud) [reply community/modules/scheduler/schedmd-slurm-gcp-v6-controller/outputs.tf]: actual output is probably longer, but it feels less error prone and I like it better. Thanks for the suggestion.

Comment (nick-stroud) [reply community/modules/scheduler/schedmd-slurm-gcp-v6-controller/outputs.tf]: ps. I implemented your suggestion if it wasn't clear from my comment.

**Pull request merged**